httpclient-jetty/src/main/java/io/fabric8/kubernetes/client/jetty/JettyHttpClientBuilder.java

```diff
@@ -39,6 +39,8 @@
     extends StandardHttpClientBuilder<JettyHttpClient, JettyHttpClientFactory, JettyHttpClientBuilder> {
 
   private static final int MAX_CONNECTIONS = Integer.MAX_VALUE;
+  // the max data in a config map is 1 MiB, so pad a little beyond that
+  private static final int MAX_WS_MESSAGE_SIZE = 1 << 21;
```

Comment:
1 MiB ? or 2 ?
> Updated the max frame and message values for jetty and vertx using 2 MB - which roughly comes from accommodating the max size of a configmap https://kubernetes.io/docs/concepts/configuration/configmap/#motivation

Reply:
With some further searching using etcd as a keyword it looks like kubernetes sets the limit on the etcd server to 3 MB per request - https://github.com/kubernetes/kubernetes/issues/105863   
https://stackoverflow.com/questions/73549236/argo-and-kubernetes-request-entity-to-large-limit-is-3145728
- rather than the etcd default of 1.5 MB.  So the 1 MB limit for a configmap is some other limit that is being enforced.  After discussion I'll update this pr to be unlimited and note in the resolution the concern about protecting the client memory from attack, which in this case would imply the api server has been compromised in some way.